tools/scripts/update_assets_metadata.py

```diff
@@ -1,28 +1,35 @@
 #!/usr/bin/env python
-import json
 import hashlib
+import json
 import re
+import sys
+from pathlib import Path
 
-ASSETS_JSON = './rotkehlchen/data/all_assets.json'
-ASSETS_META = './rotkehlchen/data/all_assets.meta'
-ASSETS_TEST = './rotkehlchen/tests/unit/test_assets.py'
+src_dir = Path(__file__).parent.parent.parent / 'rotkehlchen'
+ASSETS_JSON = src_dir / 'data/all_assets.json'
+ASSETS_META = src_dir / 'data/all_assets.meta'
+ASSETS_TEST = src_dir / 'tests/unit/test_assets.py'
 
-with open(ASSETS_META, "r") as assets_meta:
+with open(ASSETS_META, 'r') as assets_meta:
     old_meta = json.load(assets_meta)
 
-with open(ASSETS_JSON, "rb") as assets_json:
+with open(ASSETS_JSON, 'rb') as assets_json:
     new_md5 = hashlib.new('md5')
     new_md5.update(assets_json.read())
 
+if new_md5.hexdigest() == old_meta['md5']:
+    print("Assets meta file is up-to-date.")
```

Comment:
old habits die hard eh? :)

```suggestion
    print('Assets meta file is up-to-date.')
```